docs/.vuepress/config.js

```diff
@@ -186,6 +186,21 @@ module.exports = {
                       'build/examples/network-inspector/step-5-shut-down-the-application',
                       'build/examples/network-inspector/summary'
                     ]
+                  },
+                  {
+                    title: 'Simple Pinning Service',
+                    sidebarDepth: 2,
+                    collapsable: false,
+                    children: [
+                      'build/examples/simple-pinning-service/overview.md',
+                      'build/examples/simple-pinning-service/powergate-lotus-go-ipfs-interactions.md',
+                      'build/examples/simple-pinning-service/step-1-powergate-setup.md',
+                      'build/examples/simple-pinning-service/step-2-react-app-setup.md',
+                      'build/examples/simple-pinning-service/step-3-connecting-powergate-to-app.md',
+                      'build/examples/simple-pinning-service/step-4-explore-pinning-service-app.md',
+                      'build/examples/simple-pinning-service/step-5-shut-down-the-application.md',
+                      'docs/build/examples/simple-pinning-service/summary.md'
```

Comment:
It looks like these sidebar items are breaking the build. If you run `npm run docs:build` locally you will see the log output errors. Removing the .md extension here should fix it.

Reply:
Thanks!